### PR TITLE
Clarify slow start+locality weighted lb

### LINKED ISSUE #29008: Slow start mode "expires" when more members added to the cluster.
*Title*: *Slow start mode "expires" when more members added to the cluster.*

*Description*:
> When a new member joins the cluster and slow start is configured the current pod which still should be in slow start are getting all the request that they would as if the slow start window finished already. One picture is worth a thousand words so please see the attached screenshot:

Screenshot 2023-08-14 at 16 57 57

You can see that the red line starts gradually which is what I'd expect from a slow start however once the new pod (member) is added (the green line) the amount of requests directed to the red line suddenly jumps. The slow starts is configured for 300s but the "jolt" happens within a bit more than one minute from the start.

*Repro steps*:
> kubectl scale --replicas=2 deploy/aiohttp-service; sleep 75; kubectl scale --replicas=3 deploy/aiohttp-service

```
   "ISTIO_VERSION": "1.17.3",
   "version": "43cd81989c499d2382fd80ac077b14d728fe41eb/1.25.7-dev/Clean/RELEASE/BoringSSL",
```

edit: fixed my english a bit

### PR DESCRIPTION
**Pull request by nezdolik**
Clarify how slow start works with weighted locality load balancing during cross zone routing. This came out as TODO from #29008.

Commit Message:
Additional Description:
Risk Level: Low
Testing: 
Docs Changes: Done
Release Notes:
Platform Specific Features:
Fixes #29008

### CODE REVIEW

**Comment by nezdolik** [comment]
cc @wbpcode @adisuissa

File: docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst [hunk 1]
```diff
@@ -63,3 +63,4 @@ Once endpoints E1 and E2 exit slow start mode, their load balancing weight remai
    :align: center
 
 *Note* in case when multiple priorities are used with slow start and lower priority has just one endpoint A, during cross-priority spillover there will be no progressive increase of traffic to endpoint A, all traffic will shift at once.
+Same applies to locality weighted loadbalancing, when slow start is enabled for the upstream cluster and traffic is routed crozz zone to a zone with one endpoint A, there will be no no progressive increase of traffic to endpoint A.
```
Comment (phlax) [inline docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst:66]:
```suggestion
Same applies to locality weighted loadbalancing, when slow start is enabled for the upstream cluster and traffic is routed cross zone to a zone with one endpoint A, there will be no no progressive increase of traffic to endpoint A.
```

Comment (adisuissa) [reply docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst:66]: Another small nit:
s/no no/no/

Comment (nezdolik) [reply docs/root/intro/arch_overview/upstream/load_balancing/slow_start.rst]: oh, i was in a rush :) Thanks!

**Comment by nezdolik** [comment]
FYI @palmobar

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**